manifests/init.pp

```diff
@@ -72,6 +72,7 @@
   $qpid_router_broker_port   = $capsule::params::qpid_router_broker_port,
 ) inherits capsule::params {
 
+  include ::apache
```

Comment:
if i do that i need to also include those two lines in config.pp (as some config files there also reference apache_version).  Which is preferred?